documentation/api/guides/debugging.md

```diff
@@ -69,3 +69,13 @@ This is the guide to debugging the API using VS Code. This uses Microsoft's
    [Visual Code debugger's official instructions](https://code.visualstudio.com/docs/editor/debugging)
    to better understand how to use the debugger interface to accomplish tasks
    like setting breakpoints, watching variables etc.
+
+## Debugging Database Queries
+
+To log all database queries, add the following variables to your `api/.env`
+file:
+
+```
```

Comment:
All `.env` files we use are strict subsets of the shell syntax. This is also [recommended by GitHub linguist](https://github.com/github-linguist/linguist/pull/4922).

```suggestion
```shell
```

Reply:
@dhruvkb Thank you for the review, I've applied your suggestions and pushed the changes. Please review them when you can.